doc/projects.md

```diff
@@ -35,6 +35,7 @@ limits.networks                      | integer   | -                     | -
 limits.processes                     | integer   | -                     | -                         | Maximum value for the sum of individual "limits.processes" configs set on the instances of the project
 limits.virtual-machines              | integer   | -                     | -                         | Maximum number of VMs that can be created in the project
 restricted                           | boolean   | -                     | false                     | Block access to security-sensitive features
+restricted.cluster.target            | string    | -                     | block                     | Prevents from using --target to specify what cluster member to place a workload on or the ability to move a workload between members.
```

Comment:
Similar to the other two cases, I'd just make this `Prevents direct targeting of cluster members when creating or moving instances.`